go.mod

```diff
@@ -1,7 +1,9 @@
 module github.com/nats-io/nats.go
 
 require (
+	github.com/golang/protobuf v1.3.2
 	github.com/nats-io/jwt v0.3.0
+	github.com/nats-io/nats-server/v2 v2.1.0
```

Comment:
So far we have avoided adding the server deps because it is just for test. If we start to do the same on the server side, they will both depend on each other and so a project that pull nats.go say at 1.9.2 would need server v2.1.0 but if later we have server v2.1.1 that pulls nats.go v1.9.1, ... I don't know, my head spins already..

Reply:
Sorry, last commit pulled it in.

Reply:
Will revert.